### ISSUE TITLE
label-printer: the docker container dies

### ISSUE THREAD

**Issue by tingox** [bitraf/infrastructure#177]
the docker container dies when you try to print, probably something changed in Python.

**Comment by tingox** [comment]
docker logs
```
tingo@heim:~$ sudo docker logs p2k16-label-server_p2k16-label_1
deprecation warning: brother_ql.devicedependent is deprecated and will be removed in a future release
Traceback (most recent call last):
  File "/home/app_user/print_label_service.py", line 75, in <module>
    client.loop_forever()
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 1756, in loop_forever
    rc = self._loop(timeout)
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 1164, in _loop
    rc = self.loop_read()
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 1556, in loop_read
    rc = self._packet_read()
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 2439, in _packet_read
    rc = self._packet_handle()
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 3033, in _packet_handle
    return self._handle_publish()
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 3327, in _handle_publish
    self._handle_on_message(message)
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 3570, in _handle_on_message
    on_message(self, self._userdata, message)
  File "/home/app_user/print_label_service.py", line 67, in on_message
    print_label()
  File "/home/app_user/print_label_service.py", line 58, in print_label
    instructions = convert(qlr=qlr, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/brother_ql/conversion.py", line 112, in convert
    im = im.resize((dots_printable[0], hsize), Image.ANTIALIAS)
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

**Comment by tingox** [comment]
Fixed by bitraf/bitraf-iot#21

**Comment by tingox** [comment]
Fixed. Closing this.
